src/PowerModels.jl

```diff
@@ -84,6 +85,22 @@ include("util/flow_limit_cuts.jl")
 # this must come last to support automated export
 include("core/export.jl")
 
+PrecompileTools.@setup_workload begin
```

Comment:
@odow you should turn off the logger in this so that it doesn't print the logs when precompiling. It throws users off

Reply:
Hmm okay. Will do. I just assumed people ignored the stuff that happens during precompilation.